.dockerignore

```diff
@@ -0,0 +1,18 @@
+storage/
+metadata_data/
+.dockerignore
+# Byte-compiled / optimized / DLL files
+__pycache__/
+**/__pycache__
+**/*__pycache__
+**/*__pycache__*
+**/*__pycache__/
+**/__pycache__/
+*.pyc
+
+database/*
+database
```

Comment:
What is this `database` directory? Is it specific to your workflow? I don't see anything in the codebase that would result in data getting stored here.